docs/csharp/programming-guide/main-and-command-args/index.md

```diff
@@ -21,6 +21,12 @@ There can only be one entry point in a C# program. If you have more than one cla
 
 [!code-csharp[csProgGuideMain#17](~/samples/snippets/csharp/VS_Snippets_VBCSharp/csProgGuideMain/CS/Class1.cs#17)]
 
+Starting in C# 9, you can omit the `Main` method, and write C# statements as if they were in the `Main` method, as in the following example:
+
+:::code language="csharp" source="snippets/top-level-statements-1/Program.cs":::
+
+For information about how to write application code without a `Main` method, see [Top-level statements](top-level-statements.md).
```

Comment:
"with an implicit" may be more accurate?

Reply:
Agreed, that may be clearer, I'll change it.